src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import Card from "../Card/Card"
+import UniqueRandomArray from "unique-random-array"
+
+class CardContainer extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            dogs: []
+        }
+    }
+
+    componentDidMount() {
+        this.getDogs()
+    }
+
+    getDogs = () => {
+        fetch('http://localhost:3000/dogs', {
+            method: 'get'
+        })
+            .then((data) => data.json())
+            .then((res) => {
+                if (res.success) {
+                    return res.data
+                } else {
+                    return res.message
+                }
+            })
+            .then((res) => {
+                    let random = UniqueRandomArray(res)
+                    this.setState({
+                        dogs: [random(), random()]
+                    })
+                }
+            )
+    }
+
+    render() {
+        return (
+            <div className="card-container">
+                {
+                    this.state.dogs.map((dog) => {
+                        return <Card name={dog.name} height={dog.height.metric} temperament={dog.temperament}/>
```

Comment:
Good practice when we're mapping/foreach-ing over an array to produce components/elements in React is to pass down a key prop (won't make much difference with only the 2 components, but yeah good practice -- this was a comment previously and was marked as resolved but I can't see the change?

https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
on it